src/main/java/com/facebook/openwifirrm/modules/ProvMonitor.java

```diff
@@ -142,7 +142,14 @@ protected void syncDataToProv(
 				topo.computeIfAbsent(venue, k -> new TreeSet<>());
 			zone.add(tag.serialNumber);
 		}
-		deviceDataManager.setTopology(topo);
+
+		try {
+			deviceDataManager.setTopology(topo);
+		} catch (IllegalArgumentException exc) {
```

Comment:
style: this is almost always named `e` (or `e1`, `e2`, etc. if that is already taken)